src/Elcodi/Plugin/StoreTemplateBundle/Resources/views/Pages/product-view-item.html.twig

```diff
@@ -90,5 +90,7 @@
 {% endblock content %}
 
 {% block prefooter %}
-    {% include "@StoreTemplate/Modules/_product-related.html.twig" with {'products' : related_products} %}
+    {{ render(url('store_product_related', {
+        'id': product.id
+    })) }}
```

Comment:
ok, than the only change could be replacing `url` with `path`, but this doesn't change so much